packages/website/src/index.js

```diff
@@ -53,25 +53,48 @@ export const wrapPageElement = ({ element, props }) => {
               width={['100%', 200, 250]}
               height={['auto', '100vh']}
               overflow={['auto', 'scroll']}
-              pt={4}
-              pb={[0, 4]}
+              pb={[0, 8]}
               borderRight={[0, '1px solid']}
               borderBottom={['1px solid', 0]}
               borderColor={['grays.2', 'grays.2']}
-              style={{ WebkitOverflowScrolling: 'touch' }}
+              style={{ WebkitOverflowScrolling: 'touch', overflowX: 'hidden' }}
             >
-              <Heading size={1} ml={5} mb={4} fontWeight="bold">
+              <Box py={2} px={4}>
                 <Flex alignItems="center">
-                  Radix{' '}
-                  <Badge ml={2} variant="gray">
+                  <Box>
+                    <svg
```

Comment:
Hmm, finding it a bit odd just showing the Modulz logo, then Alpha next to it.

Why did you remove Radix?

Reply:
Our customers will use their company logo. We can't expect every customer to design a separate logo for their design system. Our styleguide should be 100% Modulz generated, no special treatment.

We don't have time or resources right now to design a Radix brand, even our Modulz brand is rough.

We could remove the alpha badge though and put it elsewhere.